app/assets/stylesheets/style.scss

```diff
@@ -10615,15 +10402,22 @@ background: linear-gradient(180deg, rgba(255,252,252,0) 0%, rgba(126,109,109,0.5
   top: 50px;
 }
 .width-wide {
-  width: 600px;
+  width: 100%;
+}
+
+
+body.game-show {
+  overflow: hidden !important;
 }
```

Comment:
This was the magic fix to the annoying white space. I couldn't add it to the regular body because then the home page wouldn't scroll to the search bar.